### DISCUSSION TITLE
.NET sdk 8.0.301 Availability

### DISCUSSION THREAD

**Discussion by rabberbock** [dotnet/dotnet-docker#5533] [Q&A] [answered]
Hi,

I was wondering when the latest sdk will be available via docker. There are some trimming related issues solved in the latest release that I would like to take advantage of.

Thanks!

![image](https://github.com/dotnet/dotnet-docker/assets/3247622/1b6a1cd1-e31d-4c5f-806b-6386aebd56bc)

**Comment by lbussell** [comment, marked as answer]
There are 8.0.3xx versions of the SDK image available. I'm assuming you're asking because these aren't listed on the [Docker Hub page](https://hub.docker.com/_/microsoft-dotnet-sdk/). There's been an ongoing issue of our content not being replicated to Docker Hub which is why it's out of date; [this is being worked on](https://github.com/dotnet/dotnet-docker/issues/5327). For now, you can see the latest supported tags either at the [MAR portal](https://mcr.microsoft.com/product/dotnet/sdk/tags) or [here in this repo](https://github.com/dotnet/dotnet-docker/blob/main/README.sdk.md).

**Reply by rabberbock** [reply]
@lbussell I see, thanks for your help!
